weareinreach/inreach-catalog
# adds mexico logo

## Description

Adds new logo for mexico catalog

- Asana ticket: https://app.asana.com/0/1132189118126148/1198518309132511

## PR Checklist




## How to Test

1. Navigate to AC catalog homepage
2. Select 'Mexico' from the country dropdown
3. Verify that the logo displayed in the navigation bar is the unique Mexico logo

<img width="1268" alt="image" src="https://user-images.githubusercontent.com/7406914/97432417-4e50c300-18f2-11eb-9817-629c3a6ee46e.png">
